src/TrackElementManager.jsx

```diff
@@ -59,14 +55,17 @@ export default class TrackElementManager extends React.Component {
         </div>
         <div className="form-group">
             <label>Content</label><br />
-            <textarea style={{'display': displayTextarea}}
+            <textarea style={{'display': isTextActive ? 'block' : 'none'}}
                       className="form-control"
                       value={activeElement.source}
-                      maxLength={maxLength}
+                      maxLength={140}
                       onChange={this.onTextChange.bind(this)} />
-            <div style={{'display': displayTextarea}}
-                 className="helptext">140 character limit</div>
-            <button style={{'display': displayEditSelection}}
+            <div style={{'display': isTextActive ? 'block' : 'none'}}
+                 className="helptext pull-left">140 character limit</div>
+            <div className="pull-right jux-updated-text">
```

Comment:
this had class "helptext" previously, which looked nicer. can that be added back?